PMSCore.Test/TesterTest.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace PMSCore.Test
+{
+    public class TesterTest
+    {
+        [Fact]
+        public void Constructor_InitializesATesterWithDefaultParameters()
+        {
+            // Arrange + Act
+            var tester = new Tester("Tester1");
+
+            // Assert
+            Assert.Equal("Tester1", tester.UserName);
+            Assert.Equal(Permission.TESTER, tester.GetPermission());
+            Assert.NotNull(tester); // To ensure an object is created
+        }
+
+        [Fact]
+        public void AddTestCase_AddsAValidTestCaseToList()
+        {
+            // Arrange
+            var tester = new Tester("Tester1");
+
+            // Act
+            tester.AddTestCase("Test case 1");
+
```

Comment:
Modify the Tester class to return a List<string> and validate the result directly in tests instead of relying on captured console output.